.babelrc

```diff
@@ -1,10 +1,25 @@
 {
-  "presets": ["es2015"],
-  "ignore": ["external/**/*"],
+    "presets": [
+        "es2015",
+        [
+            "env",
+            {
+                "targets": {
+                    "node": "6.9.0"
+                },
+                "loose": true,
+                //"modules": false,
+                "useBuiltIns": true,
+                "debug": false
+            }
+        ]
+    ],
+    "plugins": ["transform-runtime", "add-module-exports"],
+    "ignore": ["external/**/*"],
 
-  "env": {
-    "cjs": {
-      "plugins": ["add-module-exports"]
+    "env": {
+        "cjs": {
```

Comment:
cjs stuff is legacy, we can remove !

Reply:
Ok.  I’ll continue to work on this in the evening when the kids are in their beds. :)